dll/win32/kernel32/winnls/string/lcformat.c

```diff
@@ -604,6 +611,22 @@ static INT NLS_GetDateTimeFormatW(LCID lcid, DWORD dwFlags,
         break;
 
       case 'y':
+#ifdef __REACTOS__
+        if (IS_LCID_JAPANESE(lcid) && (dwFlags & DATE_USE_ALT_CALENDAR))
+        {
+            LPCJAPANESE_ERA pEra = JapaneseEra_Find(lpTime);
+            if (pEra)
+            {
+                if (count >= 2)
+                {
+                    count = 2;
+                }
+                dwVal = lpTime->wYear - pEra->wYear + 1;
+                szAdd = buff;
+                break;
```

Comment:
Just to confirm: the `break` belongs here, and if `pEra` is null we should execute the "normal" code below?

Reply:
Resolved as: `ERROR_ACCESS_DENIED` if the era is inaccessible or not found.

Reply:
No. It should be `ERROR_INVALID_PARAMETER`.
![invalid_era](https://user-images.githubusercontent.com/2107452/55679923-284d8180-594e-11e9-8d15-f5f82fcc051a.png)
https://jira.reactos.org/secure/attachment/52207/invalid_era.zip

Reply:
OK.